libraries/stdlib/samples/test/samples/streams/streams.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2010-2018 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package samples.streams
+
+import samples.Sample
+import samples.assertPrints
```

Comment:
Please use a star-import from `samples` package, this is a requirement of the runnable sample converter.
For the other two packages, a star-import won't hurt either.

Reply:
Thanks for the comment. I will change the samples imports to a star but when it comes to the rest I find it much easier to code review when the imports are explicit. That way you always know which classes are being used. Don't you agree?

Reply:
The way how the runnable sample converter works is that it just copies all imports from a file, no matter whether they are used in the particular sample or not. Also in a short sample where at most two-three types are used, it usually is clear where they come from.
So importing all types explicitly just adds noise to a sample.